backend/minors/ROB.yaml

```diff
@@ -0,0 +1,117 @@
+---
+type: Minor
+name: Robotics
+code: ROB
+urls:
+- https://robo.princeton.edu/minor
+description: The minor in robotics seeks to provide students from a 
+  diverse range of majors an opportunity to gain an appreciation for these 
+  challenges, develop an understanding of the current state of the art, 
+  explore the multidisciplinary and interdisciplinary nature of robotics, 
+  consider issues of societal significance, and put these lessons into 
+  practice through project-based assignments in courses and through an 
+  independent work project or thesis.
+contacts:
+- type: Director
+  name: Szymon Rusinkiewicz
+  email: smr@princeton.edu
+- type: Program Administrator
+  name: Mitra Kelly
+  email: mkelly@princeton.edu
+req_list:
+  - name: Prerequisites
+    min_needed: ALL
+    max_counted: 1
+    explanation: The minor is open to students from any major who have 
+      a satisfactory background in mathematics, physics, and computing.  
+      Students should have successfully completed or placed out of 
+      mathematics, introductory physics (including mechanics), and 
+      intoductory computer science.
+    req_list:
+    - name: Multivariable Calculus
+      max_counted: 1
+      min_needed: 2
+      explanation:
+      course_list:
+      - MAT 201
+      - MAT 202
+      - MAT 203
+      - MAT 204
+    - name: Introductory Physics
+      max_counted: 1
+      min_needed: 2
+      explanation:
+      course_list:
+      - PHY 103
+      - PHY 104
+      - PHY 105
+      - PHY 106
+    - name: Introductory Computer Science
+      max_counted: 1
+      min_needed: 1
+      explanation:
+      course_list:
+      - COS 126
+      - ECE 115
+  - name: Core
+    min_needed: 2
+    max_counted: 1
+    explanation: 
+    max_common_with_major: 0
+    course_list:
+    - MAE 345 / COS 346 / ECE 345 
+    - ECE 346 / COS 347 / MAE 346 
+    - ARC 380 
+    - CEE 374 / STC 374 
+    - ECE 302 
+    - MAE 322 
+    - MAE 412 
+    - MAE 416 / EEB 416
+  - name: Electives
+    min_needed: 3
+    max_counted: 1
+    explanation: Any of the courses in the “core” list may be counted as 
+      electives if a student takes more than two courses from that list. 
+      Additional accepted courses are listed below.
```

Comment:
We need to add the classes from Core to this list as well in case they use those to fulfill the Electives reqs